méli-mélo/méli-mélo-en.html

```diff
@@ -18,14 +18,12 @@ <h3>Méli-mélo sub-projects</h3>
 
 {% assign melimelo_pages = site.pages | where: "output", "false" | where: "feature", "méli-mélo" | sort: "componentName" %}
 
-<!--
-Number of méli-mélo: {{ melimelo_pages | size }}
--->
-
 <ul>
 {% for feature in melimelo_pages %}
-	{% assign first_example = feature.pages.examples | first %}
-	<li><strong>{{ feature.componentName }}</strong>: <a href="{{ feature.componentName }}/{{ first_example.path }}" lang="{{ first_example.language }}">{{ first_example.title }}</a><!-- {{ feature.url }} --></li>
+	{% assign featBasePath = feature.path | slice: 0, 10 %}
+	{% if featBasePath == "méli-mélo/" %}
+	<li><a href="{{ feature.componentName }}/détails-en.html">{{ feature.componentName }}</a></li>
+	{% endif %}
```

Comment:
Do you think you could also fix the link on this page at line 86, by removing the following part from it: `/gcweb-compiled-demos`.

Reply:
Yes its probably feasible to do a similar fix, but gcweb-compiled-demos is a different github reposirty

Reply:
At line 86, it refer to a different github reposority where compilated demos are published. Those compiled méli-mélo demos can only obtained when the build script is run. However, we can probably apply a fix when we are running it into a jekyll local instance